Source/RunActivity/Viewer3D/RollingStock/MSTSLocomotiveViewer.cs

```diff
@@ -1135,7 +1135,7 @@ public class CabRenderer : RenderPrimitive
         private int _Location;
         private bool _isNightTexture;
         private bool HasCabLightDirectory = false;
-        public Dictionary<int, CabViewControlRenderer> ControlMap;
+        public Dictionary<(CabViewControlType, int), CabViewControlRenderer> ControlMap;
```

Comment:
Shouldn't CabViewControlType (which is a ref to an object) be a CABViewControlTypes enum value ?

Reply:
Ok, I was a bit confused by it. Ok.